client/src/App.js

```diff
@@ -21,23 +21,16 @@ class App extends Component {
   displayPanopto(currentPanoptoIndex){
 	  this.setState({currentPanopto:this.state.urls[currentPanoptoIndex]});
   }
-  timeConverter(t) {
-	var minutes = Math.floor(t / 60);
-	var seconds = t - (minutes * 60);
   
-	if (seconds < 10) {
-	  seconds = "0" + seconds;
-	}
-  
-	if (minutes === 0) {
-	  minutes = "00";
-	}
-	else if (minutes < 10) {
-	  minutes = "0" + minutes;
-	}
-  
-	return minutes + ":" + seconds;
-  }
+ secondsToHms(d) {
+    d = Number(d);
+
+    var h = Math.floor(d / 3600);
+    var m = Math.floor(d % 3600 / 60);
+    var s = Math.floor(d % 3600 % 60);
+
+    return ("0" + h).slice(-2) + ":" + ("0" + m).slice(-2) + ":" + ("0" + s).slice(-2);
```

Comment:
You should add a comment explaining this line, since it's not immediately obvious what you are doing